blocks/pricing-matrix/pricing-matrix.css

```diff
@@ -0,0 +1,327 @@
+.pricing-matrix .quantity-selector input.quantity-range::-ms-fill-upper {
+    background: #fff
+}
+
+.pricing-matrix .quantity-selector input.quantity-range::-ms-fill-lower {
+    background-color: var(--accent-color)
+}
+
+.pricing-matrix .quantity-selector input.quantity-range::-ms-thumb {
+    background-color: var(--accent-color);
+    border: none;
+    border-radius: 50%;
+    box-shadow: 0 0 8px 0 rgb(0 0 0 / 18%);
+    height: 35px;
+    margin-top: 0;
+    width: 35px
+}
+
+
+.pricing-matrix .quantity-selector {
+    max-width: 61.8rem;
+    margin: 0 auto;
+}
+
+.pricing-matrix .quantity-selector .range-length {
+    display: flex;
+    justify-content: space-between;
+    font-size: 1.2rem;
+    font-weight: 400;
+    line-height: 1.6rem;
+    color: var(--accent-color);
+}
+
+.pricing-matrix .quantity-selector .ranges {
+    display: flex;
+    justify-content: space-evenly;
+    font-size: 1.2rem;
+    font-weight: 400;
+    letter-spacing: .025rem;
+    line-height: 1.6rem;
+    gap: 10px;
+}
+
+.pricing-matrix .quantity-selector .ranges .divider {
+    border-right: 1px solid var(--accent-color);
+    height: 0.5rem;
+    width: 0.1rem;
+}
+
+@media (max-width: 767px) {
+    .pricing-matrix .quantity-selector {
+        display: flex;
+        flex-direction: row-reverse;
+        gap: 20px;
+    }
+
+    .pricing-matrix .quantity-selector .ranges {
+        display: none;
+    }
+
+    .pricing-matrix .quantity-selector .range-length {
+        display: none;
+    }
+
+    .pricing-matrix .quantity-selector .widget-range  {
+        flex: 1;
+    }
+}
+
+
+.pricing-matrix .plans {
+    margin-top: 8rem;
+}
+
+@media (min-width: 990px) {
+    .pricing-matrix .plans {
+        gap: 40px;
+        display: flex;
+    }
+}
+
+.pricing-matrix .plans .plans-card {
+    background-color: white;
+    border-radius: 2.2rem;
+    margin-top: 35px;
+    padding-bottom: 3.2rem;
+    text-align: center;
+    font-size: var(--body-font-size-xxs);
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 560px;
+
+}
+
+@media (min-width: 990px) {
+    .pricing-matrix .plans .plans-card {
+        width: clamp(300px, 100%, 400px);
+    }
+}
+
+.pricing-matrix .plans .plans-card .plan-content {
+    padding: 60px;
+}
+
+@media (min-width: 990px) {
+    .pricing-matrix .plans .plans-card .plan-content {
+        padding: 32px;
+    }
+}
+
+.pricing-matrix .plans .plans-card .plan-basic-info h3 {
+    letter-spacing: -.06rem;
+    line-height: 3.2rem;
+    font-size: 3.2rem;
+    margin: 0;
+}
+
+.pricing-matrix .plans .plans-card .plan-basic-info p {
+    max-width: 70%;
+    margin: 0 auto;
+    padding-top: 1rem;
+    font-weight: 400;
+    letter-spacing: -.02rem;
+    line-height: 2rem;
+}
+
+.pricing-matrix .plans .plans-card .plan-price .button-container {
+    margin: 0;
+}
+
+.pricing-matrix .plans .plans-card .plan-price .button-container .button {
+    margin-top: 10px;
+    margin-bottom: 0;
+    width: 100%;
+}
+
```

Comment:
The clients site has a little mouse over scale up animation on the button:

Looks like you could get the same by adding:
```
.pricing-matrix .plans .plans-card .plan-price .button-container:hover {
    transform: scale(1.02);
}
```
as well as:

``` transition: transform .2s;```

to the existing styling rule you have for the button.

Reply:
I don't see such an effect on the original on https://www.vonage.com/unified-communications/pricing/

https://github.com/hlxsites/Vonage/assets/2250964/c19c28cf-1a79-4a7e-b3a5-59a9874a9cb9

Reply:
Yup super weird that one has it and one doesn't.  I would err on the side of adding the effect and assume that the lack in the pricing matrix that you looked at from the client side was an oversight on their part.

Reply:
Makes sense!